modMcf/src/org/aion/mcf/config/CfgLog.java

```diff
@@ -85,6 +105,26 @@ public String toXML() {
             xmlWriter.writeCharacters("\r\n\t");
             xmlWriter.writeStartElement("log");
             xmlWriter.writeCharacters("\r\n");
+
+            /** XML - Displays tag/entry in the config.xml */
+            xmlWriter.writeCharacters("\t\t");
+            xmlWriter.writeStartElement("log-file");
+            xmlWriter.writeCharacters(this.logFile + "");
+            xmlWriter.writeEndElement();
+            xmlWriter.writeCharacters("\r\n");
+
+            /** XML - Displays log-path in the config.xml */
+            xmlWriter.writeCharacters("\t\t");
+            xmlWriter.writeStartElement("log-path");
+            xmlWriter.writeCharacters(this.logPath + "");
+            xmlWriter.writeEndElement();
+            xmlWriter.writeCharacters("\r\n");
+
+            /** If file path is invalid, outputs message to config */
+            if (!isValidPath()) {
+                xmlWriter.writeCharacters("\t\tInvalid file path; set to default: 'log'\r\n");
```

Comment:
Should not log out statements into the config file, we should simply inform the user of the error and leave it as-is

Reply:
[config.txt](https://github.com/aionnetwork/aion/files/2028734/config.txt)

Possible for users to create a config file such as this if they enter an invalid file path